docs/anti-patterns.mdx

```diff
@@ -4,35 +4,110 @@ title: Cadence Anti-Patterns
 
 This is an opinionated list of issues that can be improved if they are found in Cadence code intended for production.
 
-## Security and Robustness
+# Security and Robustness
 
-### Events From Resources Might Not Be Unique
+## Avoid using `AuthAccount` as a function parameter
 
-A public function on a contract that creates a resource can be called by any account.
+### Problem
```

Comment:
Is it possible to provide pseudo code for attack vector and the solution as well. Very similar that you provided in other best practices.